.github/workflows/tests.yml

```diff
@@ -0,0 +1,66 @@
+# TODO: Line 20, enable python-version: 3.5
+# TODO: Line 69, enable pytest --doctest-modules
+
+name: Tests
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

Comment:
```suggestion
```
I usually run tests on all pull requests on all branches.  Is there a reason to avoid doing that?